src/main/java/tds/content/mappers/ContentsItemDocumentMapper.java

```diff
@@ -0,0 +1,292 @@
+ *
+ * Unless required under applicable law or agreed to in writing, software distributed under the
+ * License is distributed in an “AS IS” BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for specific language governing permissions
+ * and limitations under the license.
+ **************************************************************************************************/
+
+package tds.content.mappers;
+
+import AIR.Common.Utilities.Path;
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import tds.itemrenderer.data.ITSAttachment;
+import tds.itemrenderer.data.ITSContent;
+import tds.itemrenderer.data.ITSDocument;
+import tds.itemrenderer.data.ITSKeyboard;
+import tds.itemrenderer.data.ITSKeyboardKey;
+import tds.itemrenderer.data.ITSKeyboardRow;
+import tds.itemrenderer.data.ITSOption;
+import tds.itemrenderer.data.ITSOptionList;
+import tds.itemrenderer.data.ITSQTI;
+import tds.itemrenderer.data.apip.APIPAccessElement;
+import tds.itemrenderer.data.apip.APIPBraille;
+import tds.itemrenderer.data.apip.APIPBrailleCode;
+import tds.itemrenderer.data.apip.APIPContentLinkInfo;
+import tds.itemrenderer.data.apip.APIPReadAloud;
+import tds.itemrenderer.data.apip.APIPRelatedElementInfo;
+import tds.itemrenderer.data.apip.APIPXml;
+import tds.itemrenderer.data.xml.itemrelease.AccessElement;
+import tds.itemrenderer.data.xml.itemrelease.ApipAccessibility;
+import tds.itemrenderer.data.xml.itemrelease.Attachment;
+import tds.itemrenderer.data.xml.itemrelease.BrailleCode;
+import tds.itemrenderer.data.xml.itemrelease.Content;
+import tds.itemrenderer.data.xml.itemrelease.ContentLinkInfo;
+import tds.itemrenderer.data.xml.itemrelease.ItemPassage;
+import tds.itemrenderer.data.xml.itemrelease.Itemrelease;
+import tds.itemrenderer.data.xml.itemrelease.Key;
+import tds.itemrenderer.data.xml.itemrelease.Keyboard;
+import tds.itemrenderer.data.xml.itemrelease.KeyboardRow;
+import tds.itemrenderer.data.xml.itemrelease.Option;
+import tds.itemrenderer.data.xml.itemrelease.Optionlist;
+import tds.itemrenderer.data.xml.itemrelease.Qti;
+import tds.itemrenderer.data.xml.itemrelease.RelatedElementInfo;
+
+@Component
+public class ContentsItemDocumentMapper implements ItemDocumentMapper {
+    private final static String ITEMBODY_SPEC = "itemBody";
+    private final static String ITEMBODY_SPEC_ELEMENT = "<itemBody>";
+
+    @Override
+    public void mapItemDocument(final ITSDocument document, final Itemrelease itemXml) {
+        ItemPassage item = itemXml.getItemPassage();
+        for (Content content : item.getContent()) {
+            ITSContent itsContent = new ITSContent();
+            itsContent.setLanguage(content.getLanguage());
+            if (content.getQti() != null) {
+                itsContent.setQti(readQti(content.getQti()));
+            }
+            if (content.getIllustration() != null) {
```

Comment:
Is this null check needed? I see why you do it on the QTI section as there are actions taken. Does the setter below my comment do something with the null value?